feature-home/src/main/java/cm/aptoide/pt/feature_home/presentation/ViewModelProvider.kt

```diff
@@ -0,0 +1,43 @@
+package cm.aptoide.pt.feature_home.presentation
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.platform.LocalContext
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider.Factory
+import androidx.lifecycle.viewmodel.compose.viewModel
+import cm.aptoide.pt.aptoide_network.domain.UrlsCache
+import cm.aptoide.pt.feature_home.domain.BundlesUseCase
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+import kotlin.reflect.KFunction0
+
+@HiltViewModel
+class InjectionsProvider @Inject constructor(
+  val urlsCache: UrlsCache,
+  val bundlesUseCase: BundlesUseCase,
+) : ViewModel()
+
+@Composable
+fun bundlesList(key: String?): Pair<BundlesViewUiState, KFunction0<Unit>> {
```

Comment:
Better rename `key` to `context`.
Also don't make it nullable as this is is not a good key for the type-insensitive storage.